doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -905,6 +905,32 @@ Parametrized inductive types
      sort for the inductive definition and will produce a less convenient
      rule for case elimination.
 
+.. opt:: Uniform Inductive Parameters
+
+     When this option is set (it is off by default),
+     inductive definitions are abstracted over their parameters
+     before typechecking constructors, allowing to write:
+
+     .. coqtop:: all
+
+        Set Uniform Inductive Parameters.
+        Inductive list3 (A:Set) : Set :=
+        | nil3 : list3
+        | cons3 : A -> list3 -> list3.
+        Unset Uniform Inductive Parameters.
+
+     This behavior is essentially equivalent to starting a new section
+     and using `Context` to give the uniform parameters, like so:
```

Comment:
The proper way to quote a command is:

```
:cmd:`Context`
```

Reply:
Thanks. I hadn't seen `doc/sphnix/README`, that gave me a better sense of how the new documentation works. I fixed this, and also used `undo` for the examples.